packages/useful-types/package.json

```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/Shopify/quilt/blob/master/packages/useful-types/README.md",
   "dependencies": {
-    "@types/react": ">=16.4.0",
+    "@types/react": "^16.8.1",
```

Comment:
I wasn't sure about this one. Should I leave as is as well? It seems like we want this one to be a peer dependency too, no?

Reply:
IMO types must stay as dependencies, so that installing a package can never result in broken typings. Having the widest version range is very beneficial here, though, to reduce the crazy `@types` duplication that can happen.